migration-cluster-day2/charts/migration-cluster-day2/values.yaml

```diff
@@ -0,0 +1,19 @@
+mtvOperator:
+  subscription:
+    namespace: openshift-mtv
+    channel: release-v2.7 # the recent z version is 2.6.7
+    installPlanApproval: Automatic # whether the update should be installed automatically
+    source: redhat-operators # name of the catalog source
+nmstateOperator:
+  subscription:
+    namespace: openshift-nmstate
+    channel: stable
+    installPlanApproval: Automatic # whether the update should be installed automatically
+    source: redhat-operators # name of the catalog source
+lvmsOperator:
```

Comment:
Any reason why we can't use the `assisted-installer` lvm installation proccess for LVM?

Reply:
Are you sure there is an option to install LVM? I remember ODF but I didn't see LVM

Reply:
Should be:
https://github.com/openshift/assisted-service/blob/master/docs/user-guide/additional-operator-notes.md